os/board/rtl8730e/src/rtl8730e_boot.c

```diff
@@ -242,6 +242,7 @@ void board_gpio_initialize(void)
 				{PA_23, PIN_INPUT, PullNone},
 				/* PB_20 is gpio pin number for LED */
 				{PB_20, PIN_OUTPUT, PullDown},
+				{PB_22, PIN_OUTPUT, PullNone},
```

Comment:
1. It should be PIN_INPUT. (My previous comment was wrong)
2. Let me suggest the title like below
```
board/rtl8730e: set PB_22 GPIO pin to get outside wakeup event

The PB_22 pin will be registered as a wakeup source and will get a wakeup event from outside.
```